src/imp/schannel.rs

```diff
@@ -305,6 +312,22 @@ impl<S: io::Read + io::Write> io::Write for TlsStream<S> {
     }
 }
 
+// SChannel-specific verification callback for validating failed certificate checks
+pub trait TlsConnectorBuilderCallbackExt {
+    /// Callback function that determines whether certificate validation failures that occur when establishing connections
+    /// should be overridden. The closure should return Ok if the validation failure is to be overridden.
+    fn callback<F>(&mut self, callback: F) ->  Result<(), Error> 
```

Comment:
Can you rename this to `verify_callback` to match the method we're delegating to?